test/mocha/10-api.js

```diff
@@ -104,7 +104,7 @@ describe('bedrock-account-http', function bedrockAccountHttp() {
     it('should return an account', async function() {
       const {account: {id}} = accounts['alpha@example.com'];
       const {account: actor} = accounts['admin@example.com'];
-      actor["ACCOUNT_ACCESS"] = true;
+      actor['ACCOUNT_ACCESS'] = true;
```

Comment:
No quotes when unnecessary: `actor.ACCOUNT_ACCESS`

Reply:
That being said, this doesn't test role-based permissions... so we should also test that. The actor should be given a `sysResourceRole` that has the `ACCOUNT_ACCESS` permission. We'll want to duplicate this all the way down as well. I realize this is confusing ... and that's because our authorization framework is currently a hybrid of capabilities and role-based permissions. Have you looked at the examples here and read the "how it works" section?:

https://github.com/digitalbazaar/bedrock-permission#quick-examples

Reply:
totally no understanding of the role system right now beyond that if no id it uses what is passed in and that capabilities are appended to the actor object.

Reply:
Made this an issue here:

https://github.com/digitalbazaar/bedrock-account-http/issues/12